starter_templates/redis/nodejs/app/main.js

```diff
@@ -2,7 +2,8 @@ const net = require('net');
 
 console.log("Implement your program here!");
 
-/* Comment out the code below to pass the first stage */
+// Uncomment this block to pass the first stage
+
```

Comment:
@FastestCoderOnEarth fyi, the commenting logic expects a contiguous block of comments with no empty lines in between (this allows us to add more commented code towards the bottom if needed)